client/my-sites/sidebar/manage-menu.jsx

```diff
@@ -120,8 +121,27 @@ class ManageMenu extends PureComponent {
 				showOnAllMySites: false,
 			},
 		];
+	}
+
+	getPluginItem() {
+		const { isAtomicSite, translate } = this.props;
 
-		return items;
+		return {
+			name: 'plugins',
+			label: translate( 'Plugins' ),
+			capability: 'manage_options',
+			queryable: ! isAtomicSite,
+			config: 'manage/plugins',
+			link: '/plugins',
+			paths: [ '/extensions', '/plugins' ],
+			wpAdminLink: 'plugin-install.php?calypsoify=1',
+			showOnAllMySites: true,
+			buttonLink: ! isAtomicSite ? '/plugins/manage' + this.props.siteSlug : '',
```

Comment:
❌ Missing the slash here. I would suggest temperate literals here to avoid bugs like this.

`/plugins/manage/${ this.props.siteSlug }`

Reply:
Rebase issue. My bad.